lib/data.js

```diff
@@ -202,6 +202,30 @@ const merge = (
   return [...unique];
 };
 
+const mergeObjects = (
+  // Merge multiple object with merger
+  merger,
+  ...objs // array of objects
+  // Returns: object
+) => {
+  const keys = new Set();
+  for (const obj of objs) {
+    for (const key in obj) {
+      keys.add(key);
+    }
+  }
+
+  const result = {};
+  for (const key of keys) {
+    const args = new Array(objs.length);
+    for (let i = 0; i < objs.length; ++i) {
+      args[i] = objs[i][key];
```

Comment:
What if `objs[i][key]` will be `undefined`?

Reply:
Should all objects have all keys?

Reply:
First one - yeah will fix it
Second - no, it will just be undefined, that is expected, the merger function has to deal with that.